sdcm/remote.py

```diff
@@ -148,7 +148,8 @@ def run(self, cmd, ignore_status=False, sudo=False, verbose=True, timeout=300):
                                            encoding='utf-8',
                                            hide=True,
                                            watchers=watchers,
-                                           command_timeout=timeout)
+                                           command_timeout=timeout,
```

Comment:
would be good to test other code using local runner, to see this isn't breaking it

Reply:
I will run test-provision

Reply:
Test provision succeeded